reviewrot/githubstack.py

```diff
@@ -123,16 +129,62 @@ def get_reviews(self, uname, repo_name, state_=None,
                 log.debug("review request '%s' is not %s than specified"
                           " time interval", pr.title, state_)
                 continue
+
+            if last_comment and last_commented:
+                if self.has_new_comments(last_comment.created_at, last_commented):
+                    log.debug("merge request '%s' has new comments  in last %s days", pr.title, last_commented)
+                    continue
```

Comment:
Redundant `continue`

Reply:
We want to avoid adding pull request that had new comment, that's why the continue is used when iterating over pull requests in loop.